src/index.css

```diff
@@ -222,11 +223,21 @@ img {
         overflow-x: scroll;
     } */
 
+  .container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 80%;
```

Comment:
Setting the container to a fixed width is what ultimately resolved the shifting issue. Now this container will stay the same size regardless of the content within